Libraries/Text/RCTTextViewManager.m

```diff
@@ -35,6 +35,7 @@ - (UIView *)view
 
 #pragma mark - Unified <TextInput> properties
 
+RCT_REMAP_VIEW_PROPERTY(allowFontScaling, fontAttributes.allowFontScaling, BOOL)
```

Comment:
We also have to remap other font-related props like this one.

Reply:
What props are you talking about? What should they be remapped to? It looks like you are highlighting `allowFontScaling` which is already getting remapped to `fontAttributes`.